# Patches for microb-engine
# Oleg Romashin <oleg.romashin@nokia.com>
# DP: Added manual enable/disable for xul and rdf
# Should be possible to build with RDF or XUL+RDF
Index: mozilla/configure.in
===================================================================
--- mozilla.orig/configure.in
+++ mozilla/configure.in
@@ -7471,17 +7471,18 @@
 
 AC_SUBST(MOZ_XSLT)
 
 dnl ========================================================
 dnl disable xul
 dnl ========================================================
 MOZ_ARG_DISABLE_BOOL(xul,
 [  --disable-xul           Disable XUL],
-    MOZ_XUL= )
+    MOZ_XUL=,
+    MOZ_XUL=1 )
 if test "$MOZ_XUL"; then
   AC_DEFINE(MOZ_XUL)
 else
   dnl remove extensions that require XUL
   MOZ_EXTENSIONS=`echo $MOZ_EXTENSIONS | sed -e 's/inspector//' -e 's/venkman//' -e 's/irc//' -e 's/tasks//'`
 fi
 
 AC_SUBST(MOZ_XUL)
@@ -7627,17 +7628,18 @@
   AC_DEFINE(MOZ_PROFILELOCKING)
 fi
 
 dnl ========================================================
 dnl disable rdf services
 dnl ========================================================
 MOZ_ARG_DISABLE_BOOL(rdf,
 [  --disable-rdf           Disable RDF],
-    MOZ_RDF= )
+    MOZ_RDF=,
+    MOZ_RDF=1 )
 if test "$MOZ_RDF"; then
   AC_DEFINE(MOZ_RDF)
 fi
 
 AC_SUBST(MOZ_RDF)
 
 dnl ========================================================
 dnl necko configuration options
